app/controllers/projects/samples/clones_controller.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Projects
+  module Samples
+    # Controller actions for Project Samples Clone Controller
+    class ClonesController < Projects::ApplicationController
+      respond_to :turbo_stream
+
+      def create # rubocop:disable Metrics/AbcSize
+        new_project_id = clone_params[:new_project_id]
+        sample_ids = clone_params[:sample_ids]
+
+        @cloned_sample_ids = ::Samples::CloneService.new(@project, current_user).execute(new_project_id, sample_ids)
+
+        if @project.errors.empty?
+          render status: :ok, locals: { type: :success, message: t('.success') }
+        elsif @project.errors.include?(:sample)
```

Comment:
If only one sample id was passed in and had an error attached to the `:sample` key for the `@project.errors`, this would throw a partial content status which shouldn't be the case. Also, it might be a good idea to create a new method in this controller which will be responsible for returning the correct status and error message(s) which will be used by the `if/elsif/else` block. This should also help remove the rubocop disable

Reply:
5aa4b43e9c981feeaff46aab2d28b14e1179ba36